docs/search/criteria_reference/symbolattribute_criterion.md

```diff
@@ -0,0 +1,22 @@
+---
+description: SymbolAttribute Criterion
+---
+
+# SymbolAttributeCriterion
+
+The `SymbolAttribute` Search Criterion searches for products by symbol attribute.
+
+## Arguments
+
+- `value` - string representing the attribute value
+
+## Example
+
+### PHP
+
+``` php
+$query = new ProductQuery();
+$query->setFilter(new SymbolAttribute('ean', '5023920187205'));
```

Comment:
This example is wrong - the second argument must be an array, not a string.

Here's a corrected (and expanded) one:
``` php
<?php

use Ibexa\Contracts\ProductCatalog\Values\Product\ProductQuery;
use Ibexa\Contracts\ProductCatalogSymbolAttribute\Search\Criterion\SymbolAttribute;

$query = new ProductQuery();
$query->setFilter(new SymbolAttribute('ean', ['5023920187205']));
/** @var \Ibexa\Contracts\ProductCatalog\ProductServiceInterface $productService*/
$results = $productService->findProducts($query);
```

Please extract it to code_samples - I suggest `code_samples/back_office/search/src/Query/SymbolAttributeQuery.php`

Reply:
I was checking pages in criteria reference and we don't extract examples to code_samples.

Reply:
That's true, but we didn't have PHPStan before - now we do, and as you can see even these tiny snippets can be wrong (and PHPStan detected that, I did not have time to run the sample manually)